src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3356,3 +3356,231 @@ public abstract partial class WaitHandle : System.IDisposable
         public virtual bool WaitOne(System.TimeSpan timeout) { return default(bool); }
     }
 }
+
+namespace System.Globalization
+{
+    public enum CalendarWeekRule
+    {
+        FirstDay = 0,
+        FirstFourDayWeek = 2,
+        FirstFullWeek = 1,
+    }
+    public abstract partial class Calendar
```

Comment:
Same here, this one should already be in this file, in the other namespace declaration for System.Globalization